src/content/docs/guides/Audio/0-getting-started-with-audio.mdx

```diff
@@ -289,6 +291,115 @@ do
 CloseAllWindows();
 ```
 
+</TabItem>
+<TabItem label="Object-Oriented">
+
+```csharp
+using SplashKitSDK;
+
+namespace SoundDemoOOP
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            
+            Window window = new Window("Sound Demo", 640, 320);
+            SoundEffect sndEffect = SplashKit.LoadSoundEffect("chipmunk", "chipmunk.ogg");
+            SplashKit.LoadSoundEffect("bells", "bells.ogg");
+            SplashKit.LoadSoundEffect("camera", "camera.ogg");
+            SplashKit.LoadSoundEffect("boing", "comedy_boing.ogg");
+            SplashKit.LoadSoundEffect("dinosaur", "dinosaur.ogg");
+            SplashKit.LoadSoundEffect("bark", "dog_bark.ogg");
+
```

Comment:
The code is receiving warning errors because a line is missing, the following should be added in the next line under bark

`LoadFont("arial", "arial.ttf");`

Reply:
I have updated the code now, the code is running without warnings.